app/scripts/metamask-controller.js

```diff
@@ -2339,6 +2355,43 @@ export default class MetamaskController extends EventEmitter {
       clearPendingConfirmations.bind(this),
     );
 
+    // RemoteFeatureFlagController has subscription for preferences changes
+    this.controllerMessenger.subscribe(
+      'PreferencesController:stateChange',
+      previousValueComparator((prevState, currState) => {
+        const { useExternalServices: prevUseExternalServices } = prevState;
+        const { useExternalServices: currUseExternalServices } = currState;
+
+        console.log('-----------------');
```

Comment:
console.logs to remove

Reply:
removed in 89fe7aced2e9557d1841598bead82c5ac897cfb9